### DISCUSSION TITLE
What ports should I forward to tpot?

### DISCUSSION THREAD

**Discussion by chbeck2010** [telekom-security/tpotce#1009] [Q&A]
I didn't see. a statement on the main page... mat unfiltered means all ports? Just wondering if I should forward all port not used for my applications specifically to the tpot instance? Just curious what best practice is. Thanks

**Comment by t3chn0m4g3** [comment]
You only forward the ports you want T-Pot to reply to or you adjust T-Pot to your personal preference (adjust `/opt/tpot/etc/tpot.yml`).
The T-Pot architecture overview should keep you covered WRT ports.

**Comment by ehnwebmaster** [comment]
Ports TCP/UDP below 1-64000 for the honeypot
Ports above > 64000 only for you (admin, control panel, ssh)

List of Ports used by every honeypot:

```
cowrie --> 22 (ssh) -23 (telnet) tcp
mailoney --> 25 (smtp) tcp
snare (tanner) --> 80 tcp
citrixhoneypot --> 443 tcp
rdpy --> 3389 (rdp) tcp
dionaea --> 445 tcp (smb-netbios), 69 udp (tftp), 135 (rpc), 3306 (MySQL), 21 (ftp), 81 1433 (MSSQL), 5060 (tcp-udp, [SIP,VOIP](https://dionaea.readthedocs.io/en/latest/service/sip.html))
heralding --> 110 (pop), 143 (imap), 993 (imaps), 995 (pop3s) y 1080 (socks5), 5432 (PostgreSQL), 5900 (vnc), 465 (smtps)
adbhoney -> 5555 (tcp)
dicompot --> 11112
ciscoasa -->   5000/udp  8443 (tcp)
Conpot IEC104 -->    161 udp (snmp) y 2404
Conpot guardian_ast --> 10001
Conpot ipmi --> 623 (udp)
Conpot kamstrup_382 --> 1025 y 50100
HoneySAP --> 3299
Medpot --> 2575
ElasticPot --> 9200
```
